shared/teams/join-team/container.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import JoinTeamDialog from './'
+import {connect} from 'react-redux'
+import {compose, lifecycle, withState, withHandlers} from 'recompose'
+import {joinTeam, setTeamJoinError, setTeamJoinSuccess} from '../../actions/teams/creators'
+import {upperFirst} from 'lodash'
```

Comment:
prefer import upperFirst from 'lodash/upperFirst' so we don't pull in the entirety of lodash